pkg/utils/utils.go

```diff
@@ -718,9 +733,13 @@ type OKDClusterInfo struct {
 }
 
 func getOKDClusterInfo(r *ReconcilerBase) *OKDClusterInfo {
-	config, err := r.GetOperatorConfigMap("webconsole-config", "openshift-web-console")
+	logger := NewLogger(true)  //log in json format
```

Comment:
Should be passing the Logger from Reconcile() down to this function so that it uses the logging setting from the kAppNav CR.